x-pack/plugins/monitoring/public/views/apm/instance/index.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/*
+ * Kibana Instance
+ */
+import { get, find } from 'lodash';
+import uiRoutes from'ui/routes';
+import { ajaxErrorHandlersProvider } from 'plugins/monitoring/lib/ajax_error_handler';
+import { routeInitProvider } from 'plugins/monitoring/lib/route_init';
+import template from './index.html';
+import { timefilter } from 'ui/timefilter';
+
+function getPageData($injector) {
+  const $http = $injector.get('$http');
+  const globalState = $injector.get('globalState');
+  const $route = $injector.get('$route');
+  const url = `../api/monitoring/v1/clusters/${globalState.cluster_uuid}/apm/${$route.current.params.uuid}`;
```

Comment:
Seems like `/api/monitoring/v1/clusters/${globalState.cluster_uuid}/apm/` we use in several location. Is there a way to make this in KB a constant for apm in general that gets reused?

Reply:
Sure, not a bad idea. I don't think I saw the same pattern with beats/kibana, but we could definitely add it